### ISSUE TITLE
plugins_desc and vray_tools direcories are empty for git clone or downloaded zip

### ISSUE THREAD

**Issue by retsyo** [bdancer/blender-for-vray#137]
no matter I use "git clone https://github.com/bdancer/vb30.git vb30", or download "vb30-master.zip" directly, there is no file in plugins_desc and vray_tools. I noticed that the icons before them are different ( on https://github.com/bdancer/vb30/tree/master). 

Now I have to clone plugins_desc and vray_tools

So is this a problem? 

thanks

**Comment by bdancer** [comment]
First you need a special Blender build.
plugins_desc and vray_tools are submodules, "vb30-master.zip" will not contain them (github issue), use `git clone --recursive`

**Comment by mirlip** [comment]
it works perfect for the exporter, however, for Blender-for-Vray, after using the --recursive option, git asked me for a password for the translations in folder "locale". It failed to clone all plugins and translations.

**Comment by bdancer** [comment]
@mirlip, if you want to compile build yourself use the build script https://github.com/bdancer/vb25-patch otherwise you have to reconfigure remotes to access Blender's submodules. Use this snippet as a "howto" https://gist.github.com/bdancer/16eb4d66b44ddf0754d0 otherwise you precompiled builds at nightlies.chaosgroup.com.

**Comment by mirlip** [comment]
Yes, I compile myself. Thank you for your answer, it solves the part from #139 about the contrib addons.
